packages/amplify-graphql-types-generator/README.md

```diff
@@ -0,0 +1,226 @@
+
+
+
+
+# Amplify GraphQL types generator
```

Comment:
Is this file up to date with the latest changes to the cli?

Reply:
I think I can get rid of this all together